source/modules/values.js

```diff
@@ -0,0 +1,13 @@
+/* eslint-disable linebreak-style */
+/* eslint-disable class-methods-use-this */
+/* eslint-disable no-trailing-spaces */
```

Comment:
- [ ] Please don't disable the linters and try to fix the problem but not to disable it because the linters is important to make your code follow best practices 🚀💯🔥🙏🏻